dpe/src/commands/certify_key.rs

```diff
@@ -41,18 +41,18 @@ impl CommandExecution for CertifyKeyCmd {
         let idx = dpe.get_active_context_pos(&self.handle, locality)?;
         let context = &dpe.contexts[idx];
 
-        if self.uses_is_ca() && !dpe.support.is_ca {
+        if self.uses_is_ca() && !dpe.support.is_ca.as_bool() {
```

Comment:
Would it be cleaner to make getter functions for each of the U8Bool values. So we can do something like this instead:

```suggestion
        if self.uses_is_ca() && !dpe.support.is_ca() {
```

Reply:
Done